### ISSUE TITLE
Basic OSM POI data adapter

### ISSUE THREAD

**Issue by keunes** [eloquence/lib.reviews#52]
Inspired by #19:

I'm an active contributor to OSM and waht I'm missing there is a review system. It would be really great if lib.reviews could be used to review restaurants, public monuments, shops and other OSM POI's.

To consider:
- items should be able to be linked to both Wiki and OSM data
- "auto-sync" of address, opening hours, website, etc would be nice
- some thought would have to be put into how to handle conflicts between OSM and Wiki data

**Comment by eloquence** [comment]
Yes, this is definitely something I'd like us to build! There's been a little bit of prior discussion about this on the OSM mailing list:
https://lists.openstreetmap.org/pipermail/talk/2016-August/thread.html#76498

From that discussion, it looks like the Overpass API is the way to go. We could start without sync features by just offering a feature to select a POI and then write a review for it (which is what I'd call a "basic adapter").

If you'd like to start on this yourself, I can give you some pointers, otherwise I'll probably get to it once we have all the F-Droid integration issues knocked out.

**Comment by keunes** [comment]
(My coding knowledge is limited to basic HTML & CSS, so I'm sure I won't be of any help on implementing this ^^)

**Comment by leochras** [comment]
Any updates on this? This is one of the weakest points of OSM, so this would be a huge benefit for the project. The only similar thing I was able to find, is https://openplacereviews.org/, but it seems that it is not actively maintained anymore.

**Comment by keunes** [comment]
@leochras As you may see from the commit history, the last feature/fix commits are from two years ago. So I'm afraid that it's safe to conclude that no, there's no update on this.

**Comment by maltfield** [comment]
See also https://github.com/eloquence/lib.reviews/issues/319
